src/CSS/Home.css

```diff
@@ -1,4 +1,7 @@
-.VideoPlayer {
-    width: 50px;
-    height:auto
+
+
+.Previous {
+    content: "<-";
```

Comment:
content only works for ::after and ::before. you may as well put the content in your button